gpu_direct_rdma_access.c

```diff
@@ -838,6 +855,30 @@ struct rdma_device *rdma_open_device_server(struct sockaddr *addr)
     return NULL;
 }
 
+int rdma_reset_device(struct rdma_device *device)
+{
+    if (device->srq != NULL || !device->qpex || !device->mqpex) {
```

Comment:
I guess one of these checks will be enough, with a comment of "/* if server? */"
no need to do all checks here